zuul.d/nodeset.yaml

```diff
@@ -51,3 +51,16 @@
     groups:
       - name: computes
         nodes: []
+
+- nodeset:
+    name: centos-9-rhel-9-2-crc-extracted-2.30-3xl
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2-30-0-3xl
+      - name: standalone
+        label: cloud-rhel-9-2
+    groups:
+      - name: computes
```

Comment:
while at it, we may want to add an "ocps" group, containing the "crc" node. That would help converging I think.